projects/fizzbuzz/python/combined/fizzbuzz.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+from argparse import ArgumentParser
+
+def calc_fizzbuzz(n, rules = None):
+    string = ''
+    if rules == None:
+    # no rules were passed in, do FizzBuzzBazz
+        if n % 3 == 0:
+            string += 'Fizz'
+        if n % 5 == 0:
+            string += 'Buzz'
+        if n % 7 == 0:
+            string += 'Bazz'
+    else:
+    # some rules were passed in, use the custom rules
```

Comment:
as opposed to this, perhaps make the rules default to the 3, 5, 7 dictionary as opposed to hard coding the logic above.  Then this function will always use the rules object.

Reply:
> as opposed to this, perhaps make the rules default to the 3, 5, 7 dictionary as opposed to hard coding the logic above. Then this function will always use the rules object.

Refactored, looks much nicer now!